### ISSUE TITLE
Lab 4- Step 3 

### ISSUE THREAD

**Issue by AhmedRashwanASU** [DS4PS/cpp-526-spr-2021#8]
As per the instructions it says that we need to create a static single trend line for the graphic in orange, which will create a mess in the graph if I created the vector list with the 10 teams as below screen, so should I include the trend line for the graph or just use the dynamic trend lines for each team while selecting? 

![image](https://user-images.githubusercontent.com/77247268/106821011-d3880880-6695-11eb-8aee-531e1facb819.png)

**Comment by jamisoncrawford** [comment]
@AhmedRashwanASU the static line is the blue league average line - this is unchanging and part of the permanent backdrop of the plot.  

The orange line will represent the team average strike outs for whatever team is selected in your user input widget (the drop-down with all of your team's).  

Does this make sense?  Am I understanding correctly?

**Comment by AhmedRashwanASU** [comment]
@jamisoncrawford Yup, I might understand this wrongly, however, now the idea is clear  as below 👍 

Blue Static Line = League Average from 1900 to 2012
Orange Dynamic Line = Tend line for each selected team from the drop-down name list.

**Comment by jamisoncrawford** [comment]
@AhmedRashwanASU you got it!
